sle15/profiles/stig.profile

```diff
@@ -14,5 +14,181 @@ description: |-
     DISA STIG for SUSE Linux Enterprise 15 V1R1.
 
 selections:
+    - sshd_approved_macs=stig
```

Comment:
You can remove this variable since `sshd_use_approved_macs_ordered_stig` does not use it.